tc-package.sh

```diff
@@ -41,4 +41,15 @@ fi;
 # Make a tar of
 #  - /home/build-user/ (linux
 #  - /Users/build-user/TaskCluster/HeavyTasks/X/ (OSX)
-${TAR} -C ${DS_ROOT_TASK} ${TAR_EXCLUDE} -cf - . | pixz -9 > ${TASKCLUSTER_ARTIFACTS}/home.tar.xz
+#  - C:\builds\tc-workdir\ (windows)
+
+if [ "${OS}" = "${TC_MSYS_VERSION}" ]; then
+    export PATH=$PATH:'/c/Program\ Files/7-Zip/'
+    pushd ${DS_ROOT_TASK}
+        7z a '-xr!.\dls\' '-xr!.\tmp\' '-xr!.\msys64\' -snl -snh -so home.tar . | 7z a -si ${TASKCLUSTER_ARTIFACTS}/home.tar.xz
```

Comment:
`7z` has a weird behavior and even though we pass `-so` for outputting to `stdout`, it asks for a filename ...

Reply:
If you're using the same tool twice can't you just go straight into a .tar.xz with a single command?

Reply:
Except `7z a -ttarxz` says `unknown archive type` :[